docs/src/architecture.md

```
@@ -0,0 +1,353 @@
+  B --> C["New Incoming Traffic, scale service to 1"]
+  C --> A
+```
+
+When we enable KubeElasti on a service, the service operates in 3 modes:
+
+
+1. **Steady State**: The service is receiving traffic and doesn't need to be scaled down to 0.
+2. **Scale Down to 0**: The service hasn't received any traffic for the configured duration and can be scaled down to 0.
+3. **Scale up from 0**: The service receives traffic again and can be scaled up to the configured minTargetReplicas.
+
+<br><br>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: br

(MD033, no-inline-html)